messages/DefaultRuleManager.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
 	"warning": {
 		"targetSkipped": "Target: '%s' was not processed by any engines.",
-		"targetsSkipped": "Targets: '%s' were not processed by any engines."
+		"targetsSkipped": "Targets: '%s' were not processed by any engines.",
+		"pathDoubleProcessed": "A file was processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out our documentation on target patterns to cuztomize which paths each engine should target.",
+		"pathsDoubleProcessed": "Multiple files were processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out our documentation on target patterns to cuztomize which paths each engine should target."
```

Comment:
Should we point to the specific location in documentation?

Reply:
Maybe. If we could get a link in there, that might be nice. That said, the text is already a little long, and a link might make it overwhelming. Can we pare this down?

Reply:
Would we be able to use a link shortener and do something like: "A file was processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out the solution here: _short url to documentation_"